README.md

```diff
@@ -452,6 +472,39 @@ export default MyComponent = () => {
 
 ## Migrating between versions
 
+### Migrating from 4.x to 5.x
+
+From version 5.0 you need to pass `navigate` prop to `Security`.
+For `react-router` 5.* use `navigate={history.replace}`:
+```jsx
+import { useHistory } from 'react-router-dom';
+
+export default App = () => {
+  const history = useHistory();
+    return (
+      <Router>
+        <Security oktaAuth={oktaAuth} navigate={history.replace}>
+          // children component
+        </Security>
+      </Router>
+    );
+};
+```
+Also to use `SecureRoute` you need to pass `useOktaAuth` prop:
+```jsx
+import { useOktaAuth } from '@okta/okta-react';
+import SecureRoute from '@okta/okta-react/react-router/SecureRoute';
+
+export default App = () => (
+  <Router>
+    <Security oktaAuth={oktaAuth} navigate={history.replace} >
+      <SecureRoute useOktaAuth={useOktaAuth} path='/protected' component={Protected} />
+    </Security>
+  </Router>
+};
+```
+
+
```

Comment:
I would also add a "If you were using OktaContext directly instead of relying on SecureRoute, please note the exports have changed" or something.